smoke-test/src/test/java/org/opennms/smoketest/minion/MinionRestIT.java

```diff
@@ -80,9 +80,17 @@ public void testRestHealthServiceOnMinion() throws Exception {
         given().get("/minion/rest/health")
                 .then().assertThat()
                 .statusCode(200);
-    }
-
 
+        given().get("/minion/rest/health?tag=local")
```

Comment:
These should probably check that each of these actually has at least a subset of responses that you expect, and doesn't contain results for tags you don't expect, or it's not really validating that the different tags are doing something.

Reply:
Yeah. You are right. I'll add those checks.

Reply:
Done.